nmmo/core/observation.py

```diff
@@ -24,30 +26,46 @@ def __init__(self,
 
     entities = entities[0:config.PLAYER_N_OBS]
     entity_ids = entities[:,EntityState.State.attr_name_to_col["id"]]
+    entity_pos = entities[:,[EntityState.State.attr_name_to_col["row"],
+                             EntityState.State.attr_name_to_col["col"]]]
     self.entities = SimpleNamespace(
```

Comment:
Now that we have more functionality, how about defining classes: EntityObservations, InventoryObservations, etc... That way we can avoid defining lambdas for every observation

Reply:
Created BasicObs, EntityObs, InventoryObs and replaced SimpleNamespace.